corpus/pages/urls.py

```diff
@@ -1,9 +1,11 @@
 from django.urls import path
 
 from .views import about_us
+from .views import compsoc
 from .views import index
 
 urlpatterns = [
     path("", index, name="index"),
     path("about_us/", about_us, name="about_us"),
+    path("compsoc/", compsoc, name="compsoc"),
```

Comment:
Update the URL based on the comment in the views file.

Reply:
done